### DISCUSSION TITLE
Simpler mobile UI set-up?

### DISCUSSION THREAD

**Discussion by AniOhev** [chrisgurney/obsidian-note-toolbar#51] [Ideas]
I have a use case which I think might be common. Desktop = icon & name & hover over. Mobile = icon only.

At the moment this requires setting up two near identical entries for every toolbar item. Maybe in addition to a (1) hide mobile, and (2) hide desktop switch, there could be an extra  switch (3) "icon, no name" to save repetitious set up for mobile when we just want an icon on mobile.

**Comment by chrisgurney** [comment]
Nice idea! Thank you!

**Comment by AniOhev** [comment]
Just re-read my post. Sorry, it's not clear, I mean't a third option that will render the name & icon & hover for desktop AND the same entry just the icon on mobile. I suspect you understood despite my poor explanation. Sorry.

**Reply by chrisgurney** [reply]
All good, understood. 

This is immediately doable with a CSS snippet, which I can add to the docs. I've been meaning to add some sample snippets as they come up.

**Comment by chrisgurney** [comment]
@AniOhev So I'm starting to realize that this idea is actually necessary to implement a couple other enhancements, including sub-menus and better mobile support. 👍

**Reply by AniOhev** [reply]
More luck than knowledge, on my part! Thank you for all you are doing to develop this great plugin

**Comment by chrisgurney** [comment]
@AniOhev Check out the beta for the ability to toggle visibility of icons and labels by platform, floating buttons, and more for mobile!

[1.7 Announcement](https://github.com/chrisgurney/obsidian-note-toolbar/discussions/62)

**Comment by chrisgurney** [comment]
@AniOhev The ability to toggle visibility of icons and labels by platform, floating buttons, and more for mobile is now available in the [1.7 release](https://github.com/chrisgurney/obsidian-note-toolbar/releases/tag/1.7.21), available in Community Plugins. Thanks again for the idea!
